crates/rolldown/src/bundler.rs

```diff
@@ -106,6 +108,11 @@ impl Bundler {
     self.closed = true;
     self.plugin_driver.close_bundle().await?;
 
+    self.assets.clear();
+    self.cache = ScanStageCache::default();
+    self.file_emitter.clear();
```

Comment:
We need to clear everything stored in `Bundler` instead of relying on `Drop`.

Reply:
Not oppose this direction, but would it casue error if the bundler/assets get dropped and user try to access the assets?

This brings up a convention that if users wants to access the asset, they shouldn't make the the js object `Bundler` get GCed, which is hard thing to tell in the js world due to js doesn't have ownership concept.